tests/test_density_baselines.py

```diff
@@ -40,8 +41,8 @@ def test_density_reward(density_type, is_stationary):
   env = util.make_vec_env(env_id, 2)
 
   # construct density-based reward from expert rollouts
-  expert_trajectories_all = rollout.load_trajectories(
-    f"tests/data/rollouts/{env_id}_*.pkl")
+  glob = f"tests/data/rollouts/{env_id}_*.pkl"
```

Comment:
Hyper-pedantry: maybe `pattern` is more appropriate than `glob` (`glob` is a verb AFAIK, and is also the name of both a built-in module and the only useful function in that module).